cmd/immuclient/cli/unixcmds.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019-2020 vChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"os"
+	"os/exec"
+)
+
+var clear = map[string]func(){
```

Comment:
Please use build flag // +build windows or // +build linux  ... darwin freebsd to avoid the map

Reply:
in clear map ,i'm doing its check on the function its being used.if os supports it or not. Getting _, ok:= clear[runtime.GOOS] of it , and handling ok as this operation not supported for current OS.